picklists/AlertType.json

```diff
@@ -0,0 +1,319 @@
+{
```

Comment:
Please include only the new entries added and remove the default ones. They will be merged